nemo/utils/notebook_utils.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import glob
+import json
+import os
+import os.path
+import subprocess
+import tarfile
+from typing import Optional
+
+import librosa
```

Comment:
Nemo utils should not import librosa, it's not part of core installs. Move the utils up to common or asr collection

Reply:
I've changed to importing `sox` and locally importing rather than at the file level! Let me know if thats a reasonable solution for keeping it within utils.

Personally I think having this function is helpful, and could potentially clean up other notebooks that duplicate the same processing. There might be a better place for it to live though, let me know if that's the case